### PR TITLE
display default 403 forbidden page when there is no customization

### LINKED ISSUE #1585: Display default eforbidden page when there is no customization
In connection to #1575 when the user does not specify his own error page, some default should be displayed. Currently empty page is displayed.

### PR DESCRIPTION
**Pull request by tulinkry**
fixes #1585

### CODE REVIEW

File: web/WEB-INF/web.xml [hunk 1]
```diff
@@ -200,10 +200,6 @@
     <error-code>500</error-code>
     <location>/error</location>
   </error-page>
-  <error-page>
-      <error-code>403</error-code>
-      <location>/eforbidden</location>
-  </error-page>
```
Comment (tulinkry) [inline web/WEB-INF/web.xml]: this is necessary otherwise it would **always** trigger the customized page

File: web/WEB-INF/web.xml [hunk 2]
```diff
@@ -200,10 +200,6 @@
...
   <jsp-config>
       <jsp-property-group>
       <url-pattern>*.jsp</url-pattern>
```
Comment (vladak) [inline web/WEB-INF/web.xml]: why is this needed ?

**Comment by vladak** [comment]
what would be actually displayed with no custom config ?

File: web/WEB-INF/web.xml (hunk 2 above)
Comment (tulinkry) [inline web/WEB-INF/web.xml]: https://github.com/OpenGrok/OpenGrok/pull/1591#discussion_r116225626

File: web/eforbidden.jsp [hunk 3]
```diff
@@ -17,14 +17,15 @@ information: Portions Copyright [yyyy] [name of copyright owner]
 CDDL HEADER END
 
 Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
---%><%@page  session="false" isErrorPage="true" import="
```
Comment (vladak) [inline web/eforbidden.jsp]: why `isErrorPage` is no longer needed ?

File: web/eforbidden.jsp [hunk 4]
```diff
@@ -17,14 +17,15 @@ information: Portions Copyright [yyyy] [name of copyright owner]
...
+--%><%@page  session="false" import="
              org.opensolaris.opengrok.web.PageConfig"
 %><%
-    /* ---------------------- eforbidden.jspf start --------------------- */
-    {
+/* ---------------------- eforbidden.jspf start --------------------- */
+{
+    response.setStatus(HttpServletResponse.SC_FORBIDDEN);
 %>
 <%= PageConfig.get(request).getEnv().getConfiguration().getForbiddenIncludeFileContent()%>
```
Comment (vladak) [inline web/eforbidden.jsp]: nit: missing space at the end of function call

File: web/WEB-INF/web.xml (hunk 2 above)
Comment (vladak) [inline web/WEB-INF/web.xml]: Oh, I meant the `*.jsp` part however that is pretty standard stuff.

**Comment by tulinkry** [comment]
> what would be actually displayed with no custom config ?

screenshot from 2017-05-29 15-01-47

And the custom page if there is one.

File: web/eforbidden.jsp (hunk 4 above)
Comment (tulinkry) [inline web/eforbidden.jsp]: fixed

File: web/WEB-INF/web.xml (hunk 2 above)
Comment (tulinkry) [inline web/WEB-INF/web.xml]: that's just the way how to add the new servlet page into the request lifecycle

File: web/eforbidden.jsp (hunk 3 above)
Comment (tulinkry) [inline web/eforbidden.jsp]: I removed the definition that the eforbidden is a handler for 403 errors; in order not to confuse anybody I removed the declaration from this place as well.

This parameter in jsp means that there will be an `exception` object available for this jsp which caused the error which led to this jsp.
